### PR TITLE
docs: added provider documentation link

### PR DESCRIPTION
**Pull request by BoseSj**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
    - Added Expo Push and Firebase Cloud Messages documentation link to [index.md](https://github.com/novuhq/novu/blob/e0d8e42a6319b7e39be82c8c5dd0d28ea581c915/docs/docs/channels/push/index.md).
- **Why was this change needed?** (You can also link to an open issue here)
    - As the documentation of providers should be enlisted in [index.md](https://github.com/novuhq/novu/blob/e0d8e42a6319b7e39be82c8c5dd0d28ea581c915/docs/docs/channels/push/index.md)
- **Other information**:
_NA_

### CODE REVIEW

**Comment by BoseSj** [comment]
please review @davidsoderberg  and let me know what changes should i make to pass the tests and proceed.

File: docs/docs/channels/push/index.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
 
-Novu can be used to deliver push notifications to your customers devices. Both Mobile and Web push notifications are supported.
+Novu can be used to deliver push notifications to your customers using a unified delivery API. Both Mobile and Web push notifications are supported.
 
 You can save a user's notification/device identifiers using [Subscriber Credentials](/platform/subscribers#updating-subscriber-credentials).
+
+You can easily integrate your favorite delivery provider using the built-in [integration store](https://web.novu.co/integrations).
+
+To read a provider specific documentation:
+- [Expo Push]([url](/channels/push/expo))
```
Comment (p-fernandez) [inline docs/docs/channels/push/index.md:10]:
```suggestion
- [Expo Push](/channels/push/expo)
```

**Review by p-fernandez — CHANGES_REQUESTED** [review]

File: docs/docs/channels/push/index.md (hunk 1 above)
Comment (BoseSj) [reply docs/docs/channels/push/index.md:10]: Oops sorry for the typo.

**Comment by BoseSj** [comment]
please review changes @p-fernandez @scopsy

File: docs/docs/channels/push/index.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Push Notifications
 
-Novu can be used to deliver push notifications to your customers devices. Both Mobile and Web push notifications are supported.
+Novu can be used to deliver push notifications to your customers using a unified delivery API. Both Mobile and Web push notifications are supported.
```
Comment (p-fernandez) [inline docs/docs/channels/push/index.md:3]:
```suggestion
Novu can be used to deliver push notifications to your customers devices using a unified delivery API. Both Mobile and Web push notifications are supported.
```
Missing one word in the adding.

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**